libobs/media-io/video-io.h

```diff
@@ -33,11 +33,14 @@ typedef struct video_output video_t;
 enum video_format {
 	VIDEO_FORMAT_NONE,
 
-	/* planar 420 format */
+	/* planar 4:2:0 */
 	VIDEO_FORMAT_I420, /* three-plane */
 	VIDEO_FORMAT_NV12, /* two-plane, luma and packed chroma */
 
-	/* packed 422 formats */
+	/* planar 4:2:2 */
+	VIDEO_FORMAT_I422,
```

Comment:
VIDEO_FORMAT_I422 is part of an enum and should not be moved in that enum.

Reply:
fixed